lib/bugsnag/configuration.rb

```diff
@@ -169,19 +166,26 @@ def clear_request_data
     ##
     # Logs an info level message
     def info(message)
-      logger.info(message)
+      logger.info(format_message(message))
     end
 
     ##
     # Logs a warning level message
     def warn(message)
-      logger.warn(message)
+      logger.warn(format_message(message))
     end
 
     ##
     # Logs a debug level message
     def debug(message)
-      logger.debug(message)
+      logger.debug(format_message(message))
+    end
+
+    ##
+    # Formats a message being logged by Bugsnag
+    def format_message(message)
```

Comment:
While this is unneeded in a simpler implementation, it should be noted that this method should have been private as it has no role in the public interface.